lms/djangoapps/teams/api.py

```diff
@@ -181,6 +181,34 @@ def user_organization_protection_status(user, course_key):
 
 
 def has_specific_team_access(user, team):
+    """
+    To have access to a team a user must:
+        - Be course staff
+        OR
+        - be in the correct bubble
+        - be in the team if it is private
```

Comment:
I like the way this docstring explains what is necessary for a user to have access to a team.  Can we make the code read the same way?  Something like:
```python
return has_course_staff_privileges(user, team.course_id) or (
    _team_is_public_or_user_on_team(user, team) and _user_protection_status_matches_team(user, team)
)

def _team_is_public_or_user_on_team(user, team):
    course_module = modulestore().get_course(team.course_id)
    teamset = course_module.teams_configuration.teamsets_by_id[team.topic_id]
    if teamset.teamset_type != TeamsetType.private_managed:
        return True
    return CourseTeamMembership.is_user_on_team(user, team)
```
...and rename `valid_organization_bubble_access()` to `_user_protection_status_matches_team()` (or really any other name that doesn't include the word "bubble").